client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -19,6 +19,11 @@ import ImporterActionButtonContainer from 'my-sites/importer/importer-action-but
 import ImporterActionButton from 'my-sites/importer/importer-action-buttons/action-button';
 import ImporterCloseButton from 'my-sites/importer/importer-action-buttons/close-button';
 
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
```

Comment:
We usually don't import the same style from multiple components: it's better to create a separate stylesheets `author-mapping-item.scss` and `author-mapping-pane.scss` and move the relevant styles there. That makes for a better separation of components.

Can you do that for the importer stylesheet?